qulice-checkstyle/src/test/resources/com/qulice/checkstyle/LocalVariableNames.java

```diff
@@ -7,6 +7,7 @@
  * Simple.
  * @version $Id$
  * @author John Smith (john@example.com)
+ * @checkstyle FinalLocalVariable (51 lines)
```

Comment:
@mkordas I don't know why qulice is complaing about non final variables. So I think that there's no problem to ignore this contraint here, because we are testing variables names and not if it must be final or not. Can I continue to ignore FinalLocalVariable constraint?